x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elasticsearch/MultilingualE5SmallInternalServiceSettings.java

```diff
@@ -61,25 +126,51 @@ public static MultilingualE5SmallInternalServiceSettings.Builder fromMap(Map<Str
             }
         }
 
+        SimilarityMeasure similarity = extractSimilarity(map, ModelConfigurations.SERVICE_SETTINGS, validationException);
```

Comment:
Similarity should be cosine, if we allow the user to specify it (and align with the other model configs) then there should be a check that `similarity == cosine`

Reply:
Yeah good point, we'll just hardcode it to `cosine`.